backup.sh

```diff
@@ -70,21 +73,36 @@ do
     echo "Packing up results to $ARCHIVE_FILENAME"
     if [ $retval -eq 0 ]; then
         echo "Successfully packed backup into $ARCHIVE_FILENAME"
-        mv "$ARCHIVE_FILENAME" "$TARGETDIR"
+        cp "$ARCHIVE_FILENAME" "$TARGETDIR"
     else
         echo "Failed creating $ARCHIVE_FILENAME"
     fi
 
     rm -rf $FILES
   else
     # Move files from intermediate work to target directory
-    echo "Move backup files to target directory"
-    find $WORKDIR -type f -name "*.$CURRENT_DB.bak" -exec mv {} $TARGETDIR \;
-    find $WORKDIR -type f -name "*.$CURRENT_DB.trn" -exec mv {} $TARGETDIR \;
+    echo "Copy backup files to target directory"
+    find $WORKDIR -type f -name "*.$CURRENT_DB.bak"     -exec cp {} $TARGETDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.trn"     -exec cp {} $TARGETDIR \;
+  fi
+
+  # Push to remote directory
+  if [ "$PUSH_REMOTE_MODE" = "move" ] || [ "$PUSH_REMOTE_MODE" = "copy" ]; then
+    echo "Push backup to remote directory"
+    find $WORKDIR -type f -name "*.$CURRENT_DB.bak"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.trn"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.zip"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.tar.gz"  -exec cp {} $REMOTEDIR \;
```

Comment:
Couldn't we just also use the same find as is beeing used in the cleanup:

`find $TARGETDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;`

I think atleast we should be somewhat consistant and either just take all based on the name schema or define all file extensions everywhere.

Reply:
@pfeigl Thank you very much for your review. I am agree with both.